MicaForEveryone.Win32/Window.cs

```diff
@@ -284,8 +284,11 @@ public virtual void Dispose()
             _isDisposing = true;
             _ = NativeMethods.DestroyWindow(Handle);
             Handle = IntPtr.Zero;
-            Class.Dispose();
-            _ = NativeMethods.DestroyIcon(Class.Icon);
+            if (Class != null)
```

Comment:
I believe we should just skip Disposing if we don't have a Window Class.

Reply:
`Dispose` should be called after `Activate` method got called. We can add a flag and set it true when `Activate` called then in `Dispose` we check if this flag is true or not. calling `DestroyWindow` and `DestroyIcon` when `Handle` and `Class.Icon` are null is safe.